from src.exceptions import new_exception

class DataSelectionProcessError(new_exception.NewException):
    pass
